recipe/meta.yaml

```
@@ -21,19 +21,44 @@ requirements:
     - pyproject-metadata 0.7.1
     - python
     - setuptools
-    - tomli 1.2.2  # [py<311]
-    # mesonpy takes care of creating the wheels without using wheel. See https://github.com/mesonbuild/meson-python/blob/main/mesonpy/_wheelfile.py
+    - tomli 1.2.2       # [py<311]
```

```suggestion
    - tomli       # [py<311]
```
You can drop pinnings for python packages in host

---

I need more _science_ for which are python modules

---

It's not _science_ it's _magic_. Remembering python packages' names is magic.
Yes, the linter is misleading because it thinks that meson isn't a python package. And it's weird because it's already in the list https://github.com/anaconda/anaconda-linter/blob/7f7d7286d615d2ba10424a9a9ed6b2e34bb8a5f1/anaconda_linter/lint/check_build_help.py#L36-L53